CONTRIBUTING.md

```diff
@@ -46,3 +46,43 @@ This list is not comprehensive, meaning if you want to include more detail than
 ## Go style
 
 This will be checked automatically by our CI linter bot.
+
+## Releasing
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ If you are releasing a new kubernetes version, or any other container, +
++ make sure the versions.go has been updated accordingly. See as an      +
++ example:                                                               +
++   https://github.com/SUSE/skuba/pull/573                               +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+
+In order to create a new release, create a Pull Request with these changes:
+
+  * Update the VERSION file:
+    echo "x.y.z" > VERSION
+
+  * Commit with the message "Bump version: x.y.z", being x.y.z the version. Please use semantic versioning.
+    git commit -m "Bump version: x.y.z" -s
+
+Ask for a reviewer. Once reviewed, checkout master branch:
+
+    * git checkout master && git pull
+
+And tag:
+
+  * Tag with vx.y.z
+    git tag -s vx.y.z -m "Tag x.y.z"
+
+* Push
+  git push --tags origin master
```

Comment:
I usually do `git push upstream vx.y.z` instead; note that `origin` is usually the remote for the developer fork, whereas `upstream` is usually `github.com/suse/skuba`, but this depends a lot on every developer setup and preferences...